0-bootstrap/outputs.tf

```diff
@@ -196,6 +196,20 @@ output "cloud_build_peered_network_id" {
 #   value       = module.jenkins_bootstrap.gcs_bucket_jenkins_artifacts
 # }
 
+/* ----------------------------------------
+    Specific to gitlab_bootstrap
+   ---------------------------------------- */
+# Un-comment gitlab_bootstrap and its outputs if you want to use GitLab CI/CD instead of Cloud Build
+# output "cicd_project_id" {
+#   description = "Project where the CI/CD infrastructure for GitLab CI/CD resides."
+#   value       = module.gitlab_cicd.project_id
+# }
+
+# output "projects_gcs_bucket_tfstate" {
```

Comment:
since you moved the `module.gcp_projects_state_bucket` from the cb.tf file to to the main.tf file this output is now a general one not only for GitLab